controllers/inventoriesControllers.js

```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
+    if (allInventoryItems.length === 0) {
+      return res.status(200).json({ message: 'Currently, inventory is empty' });
+    }
+    res.status(200).json(allInventoryItems);
+  } catch (error) {
+    res.status(500).json({ error: "Failed to fetch all inventory items" });
+  }
+};
+
```

Comment:
If you check your ticket it say  **Note**: The warehouse name for each inventory item must also be included in the response. 
 i dont think you doing that